dndwiki/classes/class-gunslinger.xml

```diff
@@ -0,0 +1,874 @@
+      <grant type="Archetype Feature" name="ID_WIKI_ARCHETYPE_FEATURE_PALE_RIDER" level="20" />
+    </rules>
+  </element>
+  <element name="Spellcasting" type="Archetype Feature" source="D&amp;D Wiki" id="ID_WIKI_ARCHETYPE_FEATURE_SPELLCASTING">
+    <description>
+      <p>At 3rd level, you have learned to augment your martial prowess with divine spellcasting.</p>
+      <p class="indent"><strong><em>Cantrips.</em></strong> You can choose two cantrips from the cleric spell list. At 10th level, you can choose another cantrip from the cleric spell list.</p>
+	  <p class="indent"><strong><em>Spell Slots.</em></strong> The Preacher Spellcasting table shows how many spell slots you have to cast your spells of 1st level and higher. To cast one of these spells, you must expend a slot of the spell’s level or higher. You regain all expended spell slots when you finish a long rest.</p>
+	  <p class="indent">For example, if you know the 1st-level spell shield of faith and have a 1st-level and a 2nd-level spell slot available, you can cast shield of faith using either slot.</p>
+	  <p class="indent"><strong><em>Spells Known of 1st-Level and Higher.</em></strong> You know three 1st-level paladin spells of your choice.</p>
+      <p class="indent">The Spells Known column of the Subclass Spellcasting table shows when you learn more paladin spells of 1st level or higher. Each of these spells must be of a level for which you have spell slots. For instance, when you reach 7th level in this class, you can learn one new spell of 1st or 2nd level.</p>
+	  <p class="indent">The spells you learn at 8th, 14th, and 20th level can come from the cleric spell list.</p>
+	  <p class="indent">Charisma is your spellcasting ability for your paladin spells, since your power derives from the strength of your convictions. You use your Charisma whenever a spell refers to your spellcasting ability. In addition, you use your Charisma modifier when setting the saving throw DC for a cleric or paladin spell you cast and when making an attack roll with one.</p>
+	  <p class="indent"><strong><em>Spellcasting Ability.</em></strong> Charisma is your spellcasting ability for your wizard spells, since you learn your spells through study and memorization. You use your Charisma whenever a spell refers to your spellcasting ability. In addition, you use your Charisma modifier when setting the saving throw DC for a wizard spell you cast and when making an attack roll with one.</p>
+      <p><strong>Spell save DC</strong> = 8 + your proficiency bonus + your Charisma modifier</p>
+      <p><strong>Spell attack modifier</strong> = your proficiency bonus + your Charisma modifier</p>
+    </description>
+    <sheet>
+      <description>You can use a holy symbol as a spellcasting focus for your paladin spells.</description>
+    </sheet>
+    <spellcasting name="Preacher" ability="Charisma" prepare="false" allowReplace="true">
+      <list>Cleric, Paladin</list>
```

Comment:
Probably should do only Paladin, since you only get Cleric spells on specific levels, while Replacing allowed only for Paladin spells.